build.gradle

```diff
@@ -57,6 +61,12 @@ allprojects {
     targetCompatibility = JavaVersion.VERSION_11
   }
 
+  tasks.withType(KotlinCompile).configureEach {
+    kotlinOptions {
+      jvmTarget = "11"
```

Comment:
is this needed?

Reply:
Yes, because by default Gradle uses the JVM target of the CI, which is now 17. ideally, we should use the toolchain feature, but let's do this in another PR.